README.md

```diff
@@ -73,38 +80,46 @@ nvidia-dra-driver   nvidia-dra-plugin-lt7qh                    1/1     Running
 ### Run the examples by following the steps in the demo script
 Finally, you can run the various examples contained in the `demo/specs/quickstart` folder.
 The `README` in that directory shows the full script of the demo you can walk through.
+
 ```console
 cat demo/specs/quickstart/README.md
-...
 ```
 
-Where the running the first three examples should produce output similar to the following:
+Depending on which GPUs are available, running the first three examples will produce output similar to the following:
 ```console
-$ kubectl apply --filename=demo/specs/quickstart/gpu-test{1,2,3}.yaml
-...
-
+kubectl apply --filename=demo/specs/quickstart/gpu-test{1,2,3}.yaml
 ```
+
 ```console
-$ kubectl get pod -A
+kubectl get pod -A -l app=pod
```

Comment:
This does read well after this change. Can you update the text to first describe running the `apply` call and then running `kubectl` get to see them starting up.

Reply:
Updated it.